Common/Serialize/Serializer.h

```diff
@@ -208,27 +208,23 @@ class CChunkFileReader
 	}
 
 	template<class T>
-	static Error MeasureAndSavePtr(T &_class, u8 **saved, size_t *savedSize)
+	static Error MeasureAndSavePtr(T &_class, std::vector<u8> *saved)
 	{
 		u8 *ptr = nullptr;
 		PointerWrap p(&ptr, PointerWrap::MODE_MEASURE);
 		_class.DoState(p);
 		_assert_(p.error == PointerWrap::ERROR_NONE);
 
 		size_t measuredSize = p.Offset();
-		u8 *data = (u8 *)malloc(measuredSize);
-		if (!data)
-			return ERROR_BAD_ALLOC;
+		saved->resize(measuredSize);
```

Comment:
I guess this means we can no longer detect oom (unless exceptions enabled), which we actually had crashes before and added the above check for iirc...

-[Unknown]

Reply:
Yeah, true. I might revert to using plain allocations for that reason..

Reply:
Though, for various reasons, I'll have to remove some additional vector usages, in order to be able to use MeasureAndSavePtr still (which now can't write to a vector, so bases_[] will need to not be vectors either).